### PR TITLE
Custom settings improvements

### PR DESCRIPTION
**Pull request by minimaluminium**
- removed (default) label from the settings
- spacing improvements

### CODE REVIEW

File: ghost/admin/app/components/custom-theme-settings/select.hbs [hunk 1]
```diff
@@ -8,7 +8,7 @@
                 {{#each @setting.options as |settingOption|}}
                     <option value={{settingOption}} selected={{eq settingOption @setting.value}}>
                         {{settingOption}}
-                        {{#if (eq settingOption @setting.default)}}(default){{/if}}
+                        {{#if (eq settingOption @setting.default)}}{{/if}}
```
Comment (mike182uk) [inline ghost/admin/app/components/custom-theme-settings/select.hbs:11]:
```suggestion
```

We can probably remove this whole line now as there is nothing in the conditional?

File: ghost/admin/app/components/custom-theme-settings/boolean.hbs [hunk 2]
```diff
@@ -1,18 +1,20 @@
-<div class="gh-stack-item {{if (eq @index 0) "gh-setting-first" "gh-setting"}}">
+<div class="gh-stack-item {{if (eq @index 0) "gh-setting-first" "gh-setting"}} gh-setting-boolean">
     <div class="flex-grow-1">
-        <div class="flex justify-between items-center relative">
-            <span class="gh-setting-title mb0" for={{this.checkboxId}}>
-                {{humanize-setting-key @setting.key}}
-            </span>
-            <div class="for-switch x-small">
+        <div class="flex justify-between items-start relative">
+            <div class="flex flex-column" style="margin-right: 20px;">
```
Comment (mike182uk) [inline ghost/admin/app/components/custom-theme-settings/boolean.hbs:4]: Out of curiosity, do we not have any utility classes for the `margin-right` value that we can use? (i.e `.mr5`)

**Comment by mike182uk** [comment]
@minimaluminium Linter looks to be failing also

File: ghost/admin/app/components/custom-theme-settings/boolean.hbs (hunk 2 above)
Comment (minimaluminium) [reply ghost/admin/app/components/custom-theme-settings/boolean.hbs]: That's a good call!

**Review by mike182uk — APPROVED** [review]

**Pull request merged**